addons/a11y/README.md

```diff
@@ -53,26 +56,32 @@ import { storiesOf, addDecorator, addParameters } from '@storybook/react';
 
 import { withA11y } from '@storybook/addon-a11y';
 
-addDecorator(withA11y)
-addParameters({
-  a11y: {
-    element: '#root', // optional selector which element to inspect
-    config: {}, // axe-core configurationOptions (https://github.com/dequelabs/axe-core/blob/develop/doc/API.md#parameters-1)
-    options: {} // axe-core optionsParameter (https://github.com/dequelabs/axe-core/blob/develop/doc/API.md#options-parameter)
+export default {
```

Comment:
this code is correct to add component decorator/parameters, but isn't quite equiv to the code its replacing, which is global. intentional?

Reply:
Yes, intentional.

I don't want to repeat the entire docs/how-to-use-parameters all over the place.

I'll add just have most common use-case of parameters here.